config/wallet-config.md

```diff
@@ -0,0 +1,31 @@
+# Hiro Wallet messages
+
+All Hiro Wallet instances make HTTP requests to the file located in `config/wallet-config.json`. If there are messages, either global or pinned to a matching version, they will be displayed on the wallet's home screen.
+
+If there are no messages, the object should be empty: `{}`
+
+If there are active fiat providers, they will be display in the 'Buy' page.
+
+The schema of the `wallet-comms.json` file is validated by Github Actions.
+
+### Example
+
+A message targeting all versions is referred to with the key `global`. The JSON may look something like:
+
+```json
+{
+  "global": {
+    "messages": [
+      {
+        "purpose": "info",
+        "title": "API experiencing slowness",
+        "text": "Balances and transactions may take longer to update",
+        "learnMoreUrl": "https://stacks.co",
+        "publishedAt": "2021-09-27T07:57:37.397Z",
+        "dismissible": false
+      }
+    ],
+    "activeFiatProviders": ["transak", "okcoin"]
```

Comment:
Suggest this becomes: 

```json
"activeFiatProviders": [{ "name": "transak" }, { "name": "okcoin" }]
```

This will reduce chance of breaking API changes, if we need to add more provider metadata later. Might be better to add a "enabled" flag, rather than relying on presence to determine on/off.

It could also be a keyed object? 🤔

Reply:
Sure, I will update it and change the top level structure too.